README.md

```diff
@@ -106,25 +103,29 @@ In the Julia REPL, first load the package P4est.jl
 ```julia
 julia> using P4est
 ```
+Then load and initialize MPI
```

Comment:
Can you also load MPI.jl first? If so, it might be worth mentioning that. Otherwise, people could assume there are some required C setup calls that do not work properly.

Reply:
The order does not matter. I updated the README to be clear about this.